pkg/scheduler/dependency_graph.go

```diff
@@ -208,7 +227,7 @@ func canDependencyBelongToFlow(dep *client.Dependency, flow *client.Flow) bool {
 
 // newScheduledResource is a constructor for ScheduledResource
 func (sched Scheduler) newScheduledResource(kind, name string, resDefs map[string]client.ResourceDefinition,
-	gc *GraphContext) (*ScheduledResource, error) {
+	gc *GraphContext, silent bool) (*ScheduledResource, error) {
```

Comment:
Also the purpose of the argument is to suppress logging. Initially I wanted to make possible pass custom logger in options argument, but std log library doesn't export default logger which breaks the whole idea